src/sentry/backup/findings.py

```diff
@@ -196,9 +198,14 @@ class FindingJSONEncoder(json.JSONEncoder):
 
     def default(self, obj):
         if isinstance(obj, Finding):
-            d = deepcopy(obj.__dict__)
-            kind = d.get("kind")
+            keys = set(Finding.__annotations__.keys())
```

Comment:
curious, what was the primary motivator here, `deepcopy` was expensive?

Reply:
This is now sometimes used as a base class for a pydantic model (see: import_export/models.py), so deepcopy won't work, since __dict__ behaves unexpectedly.